tests/perfalert/test_create_alerts.py

```diff
@@ -224,3 +231,101 @@ def test_alert_change_type_absolute(test_repository,
 
     assert PerformanceAlert.objects.count() == expected_num_alerts
     assert PerformanceAlertSummary.objects.count() == expected_num_alerts
+
+
+def test_os_relevance():
+    try:
+        assert 5 == os_relevance('windows10')
+        assert 4 == os_relevance('windows7')
+        assert 3 == os_relevance('linux')
+        assert 2 == os_relevance('osx')
+        assert 1 == os_relevance('android')
+        assert 0 == os_relevance('obscure, alien OS name')
+    except KeyError:
+        pytest.fail('Provided OS does not have name/relevance defined.')
+
+
+def test_AlertSelectionConfig():
+    assert 5 == AlertSelectionConfig().max_alerts_selected
+    assert ('windows10', 'windows7', 'linux', 'osx', 'android') == AlertSelectionConfig().platforms_of_interest
+
+
+def prepare_alert_test_data():
+    alert_win10_1, alert_win10_2, \
+        alert_win7_1, alert_win7_2, \
+        alert_linux_1, alert_linux_2, \
+        alert_osx_1, alert_osx_2, \
+        alert_android_1, alert_android_2 = \
+        Mock(), Mock(), Mock(), Mock(), Mock(), Mock(), Mock(), Mock(), Mock(), Mock()
+
+    alert_win10_1.series_signature.platform.platform = 'windows10'
+    alert_win10_2.series_signature.platform.platform = 'windows10'
+    alert_win7_1.series_signature.platform.platform = 'windows7'
+    alert_win7_2.series_signature.platform.platform = 'windows7'
+    alert_linux_1.series_signature.platform.platform = 'linux'
+    alert_linux_2.series_signature.platform.platform = 'linux'
+    alert_osx_1.series_signature.platform.platform = 'osx'
+    alert_osx_2.series_signature.platform.platform = 'osx'
+    alert_android_1.series_signature.platform.platform = 'android'
+    alert_android_2.series_signature.platform.platform = 'android'
+
+    return [
+        alert_win10_1, alert_win10_2,
+        alert_win7_1, alert_win7_2,
+        alert_linux_1, alert_linux_2,
+        alert_osx_1, alert_osx_2,
+        alert_android_1, alert_android_2
+    ]
+
+
+def test_ensure_platform_variety():
+    all_sorted_alerts = prepare_alert_test_data()
+    expected_alerts_order = ('windows10', 'windows10', 'windows7', 'windows7', 'linux')
```

Comment:
Actually, this should be
```python
expected_alerts_order = ('windows10', 'windows7', 'linux', 'osx', 'android')
```
Use this test as guide when updating the `ensure_platform_variety()` function.